scripts/release/mule/package/docker/package.sh

```diff
@@ -6,12 +6,16 @@ echo
 date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
 echo
 
-ARCH_TYPE=$(./scripts/archtype.sh)
-OS_TYPE=$(./scripts/ostype.sh)
-BRANCH=${BRANCH:-$(./scripts/compute_branch.sh "$BRANCH")}
-CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
-PKG_ROOT_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+if  [ -z "$NETWORK" ]; then
+    echo "[$0] NETWORK=$NETWORK is missing."
+    exit 1
+fi
+
+CHANNEL=$("./scripts/release/mule/common/get_channel.sh" "$NETWORK")
 VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+OS_TYPE=$(uname)
+OS_TYPE=${OS_TYPE,}
+PKG_ROOT_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
```

Comment:
Why discontinue the use of `ostype.sh`? and how is `$ARCH_TYPE` determined?

Reply:
`ostype.sh`:
```
UNAME=$(uname)

if [ "${UNAME}" = "Darwin" ]; then
    echo "darwin"
elif [ "${UNAME}" = "Linux" ]; then
    echo "linux"
elif [[ "${UNAME}" == *"MINGW"* ]] || [[ ${UNAME} == *"MSYS_NT"* ]]; then
    echo "windows"
else
    echo "unsupported"
    exit 1
fi
```
Since we're not currently supporting windows, the only thing this script really helps with is to lowercase the first letter.  We could copy this into `mule`, but one of the pieces of feedback from the design meeting is to try and not copy and files.

I think this can be passed in the dockerfile, but I need to do a bit of research first to see how much evaluation the dockerfiles allow for.  I'm trying to detach as much as we can from the outside environment (re, the `scripts/` dir).

`ARCH_TYPE` is being set in the dockerfile.  See my description in the summary of this ticket.

Reply:
hmm, rather than consolidating, we are making copies increasing the amount of code that we have to manage.

Reply:
I don't think that's quite accurate.